src/modules/coreHttpApi/openapi.yml

```diff
@@ -279,6 +279,161 @@ paths:
 
   # ------------------- Attributes -------------------
 
+  /api/v2/Attributes/Own/Identity:
+    get:
+      operationId: getOwnIdentityAttributes
```

Comment:
Is there a reason why we write OwnIdentityAttributes instead of RepositoryAttributes here?

Reply:
No reason, just copy and paste

Reply:
@stnmtz do you know how the naming came about?